.gitignore

```diff
@@ -5,6 +5,7 @@
 # Django environments
 linux-venv/
 windows-env/
+env
```

Comment:
I think that one is not really needed. Just choose windows-env or linux-venv when creating a virtual environment